packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2327,6 +2329,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     }
   }
 
+  int get editableTextId => _editableTextId;
```

Comment:
here and elsewhere: missing docs

Reply:
Yeah, lots of clean up to do, lots of docs to write. For now just sending a draft. Also scheduled a VC for this Thursday to discuss if we're OK with this approach overall.